conda/plan.py

```diff
@@ -234,6 +231,11 @@ def format(s, pkg):
         for pkg in sorted(channeled):
             print(format(oldfmt[pkg] + arrow + newfmt[pkg], pkg))
 
+    if unchanneled:
```

Comment:
This is new. If none of the packages in the highest-priority channel are compatible with the environment, but a lower priority channel has one, this will say so.